lib/Extractor/KLEEBuilder.cpp

```diff
@@ -136,7 +140,23 @@ ref<Expr> ExprBuilder::makeSizedArrayRead(unsigned Width, StringRef Name,
       new Array(ArrayNames.makeName(NameStr), 1, 0, 0, Expr::Int32, Width));
   ArrayVars.push_back(Origin);
 
+  std::vector<unsigned> ZeroBits, OneBits;
   UpdateList UL(Arrays.back().get(), 0);
+  if (HarvestKnownBits && Origin && Origin->K == Inst::Var) {
```

Comment:
I don't think you want to check the command line flag here, since it is only about harvesting